src/views/partials/head.ejs

```diff
@@ -8,6 +8,22 @@
 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
 <!-- CSS Files -->
 <link href="/css/material-dashboard.min.css?v=2.1.0" rel="stylesheet" />
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+  crossorigin="anonymous">
+<!-- JS Files -->
 <script type="text/javascript" src="/js/instascan.min.js"></script>
 <script type="text/javascript" src="/js/core/webrtc-adapter.js"></script>
-<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.5.0/css/all.css" integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU" crossorigin="anonymous">
\ No newline at end of file
+<script src="https://cdn.onesignal.com/sdks/OneSignalSDK.js" async=""></script>
+
+<!-- One Signal -->
+<link rel="manifest" href="/manifest.json" />
+<script>
+  var OneSignal = window.OneSignal || [];
+  OneSignal.push(function () {
+    OneSignal.init({
+      appId: "e5e24f75-c2e8-4194-9606-6cf40c34f485",
```

Comment:
I only noticed it now, but is this suppossed to be hard coded?

Reply:
The app id can be hardcoded (see [here](https://documentation.onesignal.com/docs/accounts-and-keys#section-app-id)), as long as we keep the rest api key private it is fine.

Reply:
My concer isn’t security, it’s maintainability. Because if I understand correctly, this is the Id is for your app on OneSignal which means that whoever is going to be setting up this app will need to change the id in this file rather than some central settings file and that is not ideal.

Also, what is the id in manifest.json? Is it specific to your OneSignal app as well?

Reply:
Yes, the app id is for my app on OneSignal so if someone wanted to use a different app, then they would have to change this value as well. It makes sense to pass this from an env variable to the ejs renderer.

As for the manifest.json file, this is required by OneSignal and it not unique to the app.

Reply:
So is the gcm_sender_id in manifest.json not app-specific as well?

Reply:
> So is the gcm_sender_id in manifest.json not app-specific as well?

The gsm_sender_id is not app-specific. It is used by OneSignal if you set up Firebase analytics (see [here](https://developers.google.com/web/updates/2015/03/push-notifications-on-the-open-web#why_do_i_need_a_gcm95sender95id).